openedx/features/edly/templates/edly/edx_ace/handoutchangesnotification/email/body.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}
+                    Handout of course {{ course_name }} on {{ site_name }} has been updated
+                {% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                    <table border="1">
+                        <thead>
+                            {% if old_content%}
+                                <th>Old Content</th>
+                            {% endif %}
+                            <th>New Content</th>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                {% if old_content%}
+                                    <td>{% autoescape off %}{{ old_content }}{% endautoescape %}</td>
+                                {% endif %}
+                                <td>{% autoescape off %}{{ new_content }}{% endautoescape %}</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}You can view changes here: {{ course_url }}{% endblocktrans %}
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}
+                    Thanks,
+                {% endblocktrans %}
+                <br/>
+                {% blocktrans %}
+                    LUMS ELM training team
+                {% endblocktrans %}
```

Comment:
Can we replace `LUMS ELM` with `the platform_name` variable from configuration?

Reply:
Done